src/Take.Blip.Builder/FlowLoader.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Take.Blip.Builder.Models;
+
+namespace Take.Blip.Builder
+{
+    internal class FlowLoader : IFlowLoader
+    {
+        public Task<Flow> LoadFlowAsync(FlowType flowType, Flow parentFlow, string identifier, CancellationToken cancellationToken)
+        {
+            //Used when occurs a subflow redirect.Wille be implemented in the future.
+            throw new NotImplementedException();
```

Comment:
Could you be more specific with this exception, so we can better track it if it ever occur? Something like: `Loading a subflow from the SDK is not supported.`

Reply:
done